portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400&display=swap" rel="stylesheet">
     <meta content="width=device-width, initial-scale=1" name="viewport" />
   </head>
-  <body onload="sayHello()">
+  <body onload="sayHello(), loadProjects()">
```

Comment:
Does this comma work? I would have expected a semicolon instead?

Reply:
It did work both ways, I changed it to a semicolon